test/govuk_component/taxonomy_sidebar_test.rb

```diff
@@ -26,9 +26,6 @@ def component_name
         },
       ]
     )
-
```

Comment:
Interesting finds here! @carvil looks like some tests in this file aren't actually testing anything?